src/ui-library/MenuItem.jsx

```
@@ -0,0 +1,57 @@
+const React = require('react');
+
+const css = require('classnames');
+const CustomIcon = require('./CustomIcon');
+const MaterialIcon = require('./MaterialIcon');
+
+/*
+    PROPS           type        description
+    ----------------------------------------
+    className       string
+    value           string
+    caption         string
+
+    icon            string      Material Icon name, if !!icon, customIcon won't render
+    customIcon      string      Custom Icon name
+
+    tooltip         string
+    tooltipPosition string
+    ----------------------------------------
+*/
+
+class MenuItem extends React.Component {
+    render() {
+        const { value, icon, customIcon, caption, onClick, className } = this.props;
+        return (
+            <div
+                key={this.props.key || 'menu-item'}
```

this is not needed

---

you should set key when you create MenuItem, where it actually has siblings.

---

:thinking: yeah already done here https://github.com/PeerioTechnologies/peerio-desktop/blob/ea9c88df6c6dcc12230fce2b934723d7db557fb2/src/ui-library/MenuGlobal.jsx#L32

For some reason I thought it wasn't working without the dummy value but it must have been something else that was still throwing the errors. Fixed